src/FBProfile.ts

```diff
@@ -1,22 +1,19 @@
 /**
- * @flow
  * @format
  */
 
-'use strict';
-
-const {Platform, NativeModules} = require('react-native');
+import {Platform, NativeModules} from 'react-native';
 const Profile = NativeModules.FBProfile;
 
-type ProfileMap = {
-  firstName: string,
-  lastName: string,
-  middleName: string,
-  imageURL: string,
-  linkURL: string,
-  userID: string,
-  email: string,
-  name: string,
+export type ProfileMap = {
```

Comment:
After looked into the source code of both `facebook-ios-sdk` and `facebook-android-sdk`, I think the correct type of each key is  `string | undefined | null`

Reply:
fixed